Ash.FrontEnd/src/style.less

```diff
@@ -1,55 +1,59 @@
+@import '~normalize.css';
+
 // Application wide styles go here
 @import '~variables.less';
 
 html {
-  height: 100%;
   font-family: 'lato';
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  -webkit-box-sizing: inherit;
+  -moz-box-sizing: inherit;
+  box-sizing: inherit;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 50px;
 }
 
 body {
   height: 100%;
-  margin: 0;
   background-color: @app-background-color;
 }
 
 div {
-  display: block;
   color: @app-text-color;
 }
 
 #app {
   height: 100%;
 }
 
-.ash-body {
-  height: 100%;
-  display: flex;
-  flex-direction: column;
-
-  .ash-page-container {
-    height: 100%;
-    display: flex;
-    justify-content: space-between;
-
-    .ash-sidebar-left {
-      flex: 0 0 auto;
-      width: 20%;
-      height: 100%;
-    }
-
-    .ash-page-content {
-      flex: 1 0 auto;
-      height: 100%;
-      border-left: 1px solid @app-border-color;
-      border-right: 1px solid @app-border-color;
-    }
-
-    .ash-sidebar-right {
-      flex: 0 0 auto;
-      width: 20%;
-      height: 100%;
-    }
-  }
+.ash-divider {
+  position: fixed;
+  top: @header-height;
+  left: @left-gutter-width;
+  width: calc(90vw - @left-gutter-width);
```

Comment:
There a reason the navbar is 90% and this is 90vw? Does that end up the same?

Reply:
It should end up the same.  I'll change it!